client/src/utilities/levelDescriptions.ts

```diff
@@ -0,0 +1,12 @@
+export const levelDescriptions: { [key: number]: string } = {
```

Comment:
I think this is just a placeholder but just a reminder to insert the text in english to be coherent with the rest of the application 😋